README.md

```diff
@@ -196,7 +196,48 @@ actually how [sss-cli] produces variable-length shares.)
 
 ## Questions
 
-Feel free to send me an email on my Github associated e-mail address.
+### I do not know a lot about secret sharing. Is Shamir secret sharing useful for me?
+
+It depends. In the case of threshold schemes (that's what this is) there are
```

Comment:
Since it's not mentioned elsewhere, perhaps add a single line explaining _threshold schemes_?

Reply:
Hmmm. Maybe leave out the term in total? I do not see explaining it here having a lot of benefit to the reader.

Reply:
Nevermind, we mention it in the introduction. :P